src/models/ui.ts

```diff
@@ -1,23 +1,15 @@
 import { types } from "mobx-state-tree";
 
-export type Spaces = "populations";     // eventually a union of all the spaces
+export type Spaces = "populations" | "breeding" | "organism" | "dna" | "none";
```

Comment:
Realizing this should probably just be `Space`

Reply:
Good call.  `Space` is more correct for an enum-like type such as this.

Reply:
Although wouldn't it be better to handle it like the other enums?  And do something like:
'export const SpaceTypeEnum = types.enumeration("type", ["populations", "breeding", "organism", "dna", "none"]);'
'export type SpaceType = typeof SpaceTypeEnum.Type;'

Reply:
Oh yes, this is definitely an improvement! Before this, the space type in the UI was just a string. After your change you ended up with 3 types, I think you want something like this:

```
export const Space = types.enumeration("type", ["populations", "breeding", "organism", "dna", "none"]);
export type SpaceType = typeof Space.Type;
```

Reply:
Oops!  Meant to rip that obsolete `type Space` out!  Thanks for catching that.  I prefer using the naming conventions that match the rest of the code (i.e., `SpaceTypeEnum` and `SpaceType` - although this is starting to feel very sci fi - and names are probably going to drift as we add more types).

Reply:
d'oh, good call, looks like that is our convention after all! I forgot I changed it for `Genotype` since `GenotypeType` just sounds...weird.

Reply:
I ripped out the old `type Space`.  I kept the naming convention on the others, but if sticking `Type` on the end of the exported types gets too annoying/repetitive, we can change.